app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
+    <string name="common_ok">OK</string>
```

Comment:
Some of the built in strings are a bit misleading (ex. `@android:string/yes` actually maps to OK), and they are kinda obscure to most people. I'm including these since almost every project used/will use these strings, so it's better to define they here rather than redefining them for every project.